src/metabase/search/postgres/index.clj

```diff
@@ -49,38 +49,38 @@
   (-> (sql.helpers/create-table table-name)
       (sql.helpers/with-columns
         [[:id :bigint [:primary-key] [:raw "GENERATED BY DEFAULT AS IDENTITY"]]
-        ;; entity
+         ;; entity
          [:model_id :int :not-null]
          [:model [:varchar 254] :not-null] ;; TODO We could shrink this to just what we need.
          [:name :text :not-null]
-        ;; search
+         ;; search
          [:search_vector :tsvector :not-null]
          [:with_native_query_vector :tsvector :not-null]
-        ;; results
+         ;; results
          [:display_data :text :not-null]
          [:legacy_input :text :not-null]
-        ;; scoring related
+         ;; scoring related
          [:dashboardcard_count :int]
-         [:last_viewed_at :timestamp]
+         [:last_viewed_at :timestamp-with-time-zone]
          [:official_collection :boolean]
          [:pinned :boolean]
          [:verified :boolean]
          [:view_count :int]
-        ;; permission related entities
+         ;; permission related entities
          [:collection_id :int]
          [:database_id :int]
-        ;; filter related
+         ;; filter related
          [:archived :boolean :not-null [:default false]]
          [:creator_id :int]
-         [:last_edited_at :timestamp]
+         [:last_edited_at :timestamp-with-time-zone]
```

Comment:
should all be with time zone
ref: https://github.com/metabase/metabase/issues/33577